app/jobs/clock/subscriptions_to_be_terminated_job.rb

```diff
@@ -10,8 +10,11 @@ def perform
         .joins('left join webhooks on subscriptions.id = webhooks.object_id and '\
                "webhooks.webhook_type = 'subscription.termination_alert'")
         .active
-        .where("DATE(#{Subscription.ending_at_in_timezone_sql}) = ?", (Time.current + 15.days).to_date)
-        .where('webhooks.id IS NULL')
+        .where(
+          "DATE(subscriptions.ending_at::timestamptz) IN (?)",
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)